test/suite/test_verbose03.py

```diff
@@ -0,0 +1,175 @@
+    # Validates the JSON schema of a given event handler message, ensuring the schema is consistent and expected.
+    def validate_json_schema(self, json_msg):
+        expected_schema = dict(self.expected_json_schema)
+        for field in json_msg:
+            # Assert the JSON field is valid and expected.
+            self.assertTrue(field in expected_schema, 'Unexpected field "%s" in JSON message: %s' % (field, str(json_msg)))
+            # Assert the type of the JSON field is expected.
+            self.assertEqual(type(json_msg[field]), expected_schema[field]['type'],
+                    'Unexpected type of field "%s" in JSON message, expected "%s" but got "%s": %s' % (field,
+                        str(expected_schema[field]['type']), str(type(json_msg[field])), str(json_msg)))
+            expected_schema.pop(field, None)
+        # Go through the remaining fields in the schema and ensure we've seen all the fields that are always expected be present
+        # in the JSON message
+        for field in expected_schema:
+            self.assertFalse(expected_schema[field]['always_expected'], 'Expected field "%s" in JSON message, but not found: %s' %
+                (field, str(json_msg)))
+
+    @contextmanager
+    def expect_event_handler_json(self, config, stdErr=False):
+        # Clean the stdout/stderr resource before yielding the context to the execution block. We only want to
+        # capture the verbose output of the using context (ignoring any previous output up to this point).
+        if stdErr:
+            self.cleanStderr()
+        else:
+            self.cleanStdout()
+        # Create a new connection with JSON format enabled.
+        if stdErr:
+            conn_config = 'json_output=[error]'
+        else:
+            conn_config = 'json_output=[message]'
+        if config != "":
+            conn_config += "," + config
+        conn = self.wiredtiger_open(self.home, conn_config)
+        # Yield the connection resource to the execution context, allowing it to perform any necessary
+        # operations on the connection (for generating the expected message output).
+        yield conn
+        # Read the contents of stdout/stderr to extract our messages.
+        output = self.readStderr(self.nlines) if stdErr else self.readStdout(self.nlines)
+        # Split the output into their individual messages. We want validate the contents of each message
+        # to ensure we've only generated JSON messages.
+        messages = output.splitlines()
+
+        if len(output) >= self.nlines:
+            # If we've read the maximum number of characters, its likely that the last line is truncated ('...'). In this
+            # case, trim the last message as we can't parse it.
+            messages = messages[:-1]
+
+        # Test the contents of each verbose message, ensuring we can successfully parse the JSON and that is subscribes
+        # to the expected schema.
+        for line in messages:
+            try:
```

Comment:
Is the `try/except` block required here? We could just call `json.loads(line)` and if it fails, it would raise an error.

Reply:
I specifically wanted to catch the exception to print an appropriate error message along with the malformed json string. I found having the extra error message was a bit clearer from a diagnosability perspective (the exception message string from `json.loads(line)` is not super clear)